R/tm_outliers.R

```diff
@@ -11,17 +11,46 @@
 #' Specifies variable(s) to be analyzed for outliers.
 #' @param categorical_var (`data_extract_spec` or `list` of multiple `data_extract_spec`) optional,
 #' specifies the categorical variable(s) to split the selected outlier variables on.
-#'
-#' @templateVar ggnames "Boxplot","Density Plot","Cumulative Distribution Plot"
-#' @template ggplot2_args_multi
+#' @param ggplot2_args `r roxygen_ggplot2_args_param("Boxplot", "Density Plot", "Cumulative Distribution Plot")`
+#' @param decorators `r roxygen_decorators_param("tm_outliers")`
 #'
 #' @inherit shared_params return
 #'
+#' @section Decorating `tm_outliers`:
```

Comment:
Hey, the huge app from the example does not show outputs for Boxplot and Density plot in the Outliers module.

Reply:
Sorry, false alarm! It was my browser. To see the plots I needed to zoom-in or zoom-out